lib/types/src/value.rs

```diff
@@ -43,6 +67,27 @@ macro_rules! partial_eq {
         }
     )*)
 }
+
+macro_rules! into {
+    ($($t:ty => $f:tt),*) => ($(
+        impl From<&RawValue> for $t {
+            fn from(from: &RawValue) -> $t {
+                unsafe {
+                    match &from {
+                        RawValue { i32: i32 } => *i32 as _,
+                        // RawValue { f32: f32 } => *f32 as _,
+                        // RawValue { u32: u32 } => *u32 as _,
+                        // RawValue { i64: i64 } => *i64 as _,
+                        // RawValue { f64: f64 } => *f64 as _,
+                        // RawValue { u64: u64 } => *u64 as _,
+                        // _ => unimplemented!()
+                    }
```

Comment:
This implementation is a bit of a hack, because is using a Rust Union which is inheritnly unsafe.
Basically, when we had the different values for matching the union, Rust linter was complaining that the rest of match cases were not being accessed (because Rust don't know the type of the value)